src/Elmish.WPF/Binding.fs

```diff
@@ -237,9 +240,10 @@ module Binding =
     let vopt (bindings: unit -> Binding<'model, 'msg> list)
         : string -> Binding<'model voption, 'msg> =
       { GetModel = id
-        GetBindings = bindings
+        CreateViewModel = fun args -> ViewModel<'model, 'msg>(args, bindings ())
```

Comment:
Here, as opposed to the old place they were created, you actually have all of the type information.